scripts/add-guest-user.sh

```diff
@@ -9,4 +9,4 @@ docker pull $image
 docker run --network=hint_nw \
 		-v $TEST_CONFIG:/etc/hint/config.properties \
 		$image \
-		add-user guest
+		/userCLI/bin/userCLI add-user guest
```

Comment:
Yes, this should be done via the plugin. I think it would also be preferable to sort the tagging issue in the plugin (the previous PR was merged before I had a chance to look in more detail at this new plugin, but I'll have a play with it now.)

Reply:
@hillalex  It was my fault, I forgot once you add a comment on the PR you are meant to have it approved before merging. I felt I already added a fix to your request so I merged. 😞

Reply:
Ok, so the issue with the plugin is that in the image it generates it uses `CMD` rather than `ENTRYPOINT`.  https://github.com/JetBrains/package-search-gradle-plugins/blob/master/src/main/kotlin/org/jetbrains/gradle/plugins/docker/tasks/GenerateJvmAppDockerfile.kt#L21

`CMD` gets overriden by whatever command line params passed, hence why the binary has to be specified again to execute it.

Looks like they have an open PR to change this: https://github.com/JetBrains/package-search-gradle-plugins/pull/10/files

That plugin is very minimal and badly documented and unfortunately I just cannot get the custom image build to work. I suggest using the same plugin as OrderlyWeb: 

```
plugins {
    id 'com.palantir.docker' version '0.33.0'
}
```

example setup:

https://github.com/vimc/orderly-web/blob/master/src/userCLI/build.gradle#L34

with custom Dockerfile in the root of the `userCLI` project:

https://github.com/vimc/orderly-web/blob/master/src/userCLI/Dockerfile